### ISSUE TITLE
Local environment change after upgrading to .net core 3.1.5

### ISSUE THREAD

**Issue by RiskXs** [dotnet/sdk#12068]
Hi,

I've upgrade my VS(16.6.2), SDK(3.1.301) and .NET runtime(3.1.5) and I'm noticing strange behavior.
Before the upgrade the HostingEnvironment.EnvironmentName was set to Develop and after the upgrade it was set to Production.
I had to add environment variable in order to set it to Develop. Any idea why is this happening?

**Comment by markmblc** [comment]
The behavior of my application also changed. It seems that the ASPNETCORE_ENVIRONMENT variable set in my launchSettings.json profile is being ignored when the commandName is "IISExpress". I had to switch to "Project" for now which seems to be working.

**Comment by DamianEdwards** [comment]
I'm not able to reproduce this.

My steps:
1. Update VS to 16.6.3 (latest on release channel)
1. Confirm .NET SDK is updated to 3.1.301
1. Create a new project in VS using the ASP.NET Core Web Application template using the "Web Application" content choice and accept all other defaults
1. Update the `Pages\Index.cshtml` to show the environment name, e.g.
    ``` html
    @page
    @model IndexModel
    @inject Microsoft.AspNetCore.Hosting.IWebHostEnvironment Env
    @{
        ViewData["Title"] = "Home page";
    }

        <div class="text-center">
            Running in @Env.EnvironmentName
        </div>
    ```
1. Launch the application with the debugger using the IIS Express profile (hit F5)
1. Confirm the name is "Development"
1. Stop the application
1. Launch the application again, this time without debugging, using the IIS Express profile (hit Ctrl+F5)
1. Confirm the name is "Development"

Could you confirm you're seeing this behavior after following the above, or point to a set of steps that reproduces the behavior for you?

**Comment by sayedihashimi** [comment]
I tried it as well, and it was working fine for me.

**Comment by vijayrkn** [comment]
Dupe of this issue https://github.com/dotnet/websdk/issues/1510. This was fixed and released in 16.6.3 on 6/30.
